plugins/inputs/opcua/sample.conf

```diff
@@ -48,18 +48,38 @@
   ## namespace         - OPC UA namespace of the node (integer value 0 thru 3)
   ## identifier_type   - OPC UA ID type (s=string, i=numeric, g=guid, b=opaque)
   ## identifier        - OPC UA ID (tag as shown in opcua browser)
-  ## tags              - extra tags to be added to the output metric (optional)
-  ## Example:
-  ## {name="ProductUri", namespace="0", identifier_type="i", identifier="2262", tags=[["tag1","value1"],["tag2","value2"]]}
-  # nodes = [
-  #  {name="", namespace="", identifier_type="", identifier=""},
-  #  {name="", namespace="", identifier_type="", identifier=""},
-  #]
+  ## tags              - extra tags to be added to the output metric (optional); deprecated in 1.25.0; use default_tags
+  ## default_tags      - extra tags to be added to the output metric (optional)
+  ##
+  ## Use either the inline notation or the bracketed notation, not both.
+  #
+  ## Inline notation (default_tags not supported yet)
+  # node = [
```

Comment:
should this be `nodes` as it was previously?

Reply:
Yes, that was a typo